app/src/main/java/org/fossasia/openevent/general/event/EventUtils.kt

```diff
@@ -52,15 +52,37 @@ object EventUtils {
             .toOffsetDateTime()
             .atZoneSameInstant(ZoneId.systemDefault())
 
-    fun getTimeInMilliSeconds(date: String): Long{
-        var testDate =  Date()
-        val dateString = date.replace("T"," ")
-            val dateFormat = SimpleDateFormat("yyyy-MM-dd HH:mm:ss", Locale.getDefault())
-            try {
-                testDate = dateFormat.parse(dateString)
-            } catch (e: ParseException) {
-                Timber.e(e,"Error Parsing Date")
-            }
+    fun getTimeInMilliSeconds(date: String): Long {
+        var testDate = Date()
+        val dateString = date.replace("T", " ")
```

Comment:
Wrong logic

Reply:
i havent added this part of code just refacoted it. however i will change these things